### ISSUE TITLE
Centering the table

### ISSUE THREAD

**Issue by tcf909** [bosskmk/pluto_grid#244]
Hello,

I'm having a hard time centering this table in any type of content. There seems to be a lot of forced positioning. Could you suggest a path to center PlutoGrid in another Container?

Thanks

**Comment by bosskmk** [comment]
```dart
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: Container(
          width: 500,
          height: 300,
          child: PlutoGrid(
            columns: columns,
            rows: rows,
            onChanged: (PlutoGridOnChangedEvent event) {
              print(event);
            },
          ),
        ),
      ),
    );
  }
```

![plutogrid_centering](https://user-images.githubusercontent.com/5284141/127635691-0e30c48d-d2d9-42e6-8340-028061dd817d.png)

**Comment by bosskmk** [comment]
This is just one example. You can center the plutoGrid in different ways in different situations.

**Comment by tcf909** [comment]
> This is just one example. You can center the plutoGrid in different ways in different situations.

Thanks for posting an example. Once we figured out we needed to fix a size it helped.

The other thing we haven't been able to figure out is how to make the column width more fluid. Right now the column widths are very static. We would like to be able to set minimum widths then everything beyond a minimum that didn't have a specific width set would work more like Flexible.

Any thoughts?
